komodo-parent/pom.xml

```diff
@@ -507,7 +518,55 @@
 				<version>${version.log4j}</version>
 			</dependency>
 
+			<dependency>
+				<groupId>org.jboss.spec</groupId>
+				<artifactId>jboss-javaee-6.0</artifactId>
+				<version>${version.org.jboss.spec.jboss-javaee-6.0}</version>
+				<type>pom</type>
+			</dependency>
+			<dependency>
```

Comment:
Do the `arquillian` entries in `dependencyManagment` need `<scope>test</scope>` like hamcrest, mockito, and junit have? Or maybe the `scope` element can be removed from those.